pretyping/detyping.ml

```diff
@@ -196,6 +196,14 @@ type _ delay =
 (** Should we keep details of universes during detyping ? *)
 let print_universes = ref false
 
+(** Should we print hidden sort quality variables? *)
+let print_sort_quality =
```

Comment:
Do we want to document this?

Reply:
Unclear for now, but there will be additional tweaks soon.